wsagent/che-core-git-impl-jgit/src/main/java/org/eclipse/che/git/impl/jgit/JGitConnection.java

```diff
@@ -226,6 +227,8 @@
     private static final String MESSAGE_COMMIT_NOT_POSSIBLE       = "Commit is not possible because repository state is '%s'";
     private static final String MESSAGE_COMMIT_AMEND_NOT_POSSIBLE = "Amend is not possible because repository state is '%s'";
 
```

Comment:
I think it is better to remove this constant and define the string directly in the check because it is only a part of error and it is used only one time. WDYT?